app/controllers/users/posts_controller.rb

```diff
@@ -0,0 +1,20 @@
+# app/controllers/users/posts_controller.rb
+module Users
+  class PostsController < ApplicationController
+    before_action :set_user, only: %i[index show]
+
+    def index
+      @posts = @user.posts
+    end
+
+    def show
+      @post = @user.posts.find(params[:id])
+    end
+
+    private
+
+    def set_user
+      @user = User.find(params[:user_id])
+    end
+  end
+end
```

Comment:
- [ ] You already created a `posts controller` in `./app/controllers/posts_controller.rb`, there is no need to create this one here please remove it to follow the DRY coding practice principle.

![image](https://github.com/NoelLincoln/blog-app/assets/68796232/06686897-b444-406d-be44-6fb810be1e65)